examples/batch.py

```diff
@@ -0,0 +1,169 @@
+import shippo
+import time
+
+"""
+In this tutorial we see how to use and interact with batches
+"""
+
+# Replace <API-KEY> with your key
+shippo.api_key = "<API-KEY>"
+
+example_batch = {
+    "default_carrier_account": "0d19dbd3b09544c79be2ab1b780f490d", #replace with your carrier account
```

Comment:
Can we replace this with a string that indicates that the object_id needs to be replaced